react/features/invite/components/dial-in-summary/DialInSummary.web.js

```diff
@@ -56,9 +56,9 @@ type State = {
     loading: boolean,
 
     /**
-     * The dial-in numbers. entered by the local participant.
+     * The dial-in numbers entered by the local participant.
```

Comment:
How did the local participant enter these numbers?

Reply:
I don't know, but fixed it.